app/src/main/java/com/microsoft/research/karya/ui/scenarios/imageData/ImageDataFragment.kt

```diff
@@ -246,24 +242,24 @@ class ImageDataFragment : BaseMTRendererFragment(R.layout.microtask_image_data)
         imageListAdapter.updateItem(index, path)
     }
 
-    private fun handleGridImageClick(index: Int) {
+    private fun handleGridImageClick(index: Int) = with(binding) {
         currentImageIndex = index
         if (localImageState[index]) {
             switchToFullImageDisplayView()
         } else {
             recaptureBtn.disable()
-            startCaptureBtn.disable()
+            startCaptureBtn.root.disable()
             cameraCv.open()
         }
     }
 
     private fun updateFullImageView() {
```

Comment:
Just a nitpick: you can add with(binding) here since we are referencing more than one view. You can come up with a general rule of updating code: using with keyword for the functions (or block of code) referencing more than one view else you can just get away with referencing binding object. What do you think?

Reply:
Or you can add with(binding) everywhere no matter the number of times views are referenced. Just to be consistent

Reply:
Yes I was thinking that, I would go with the `with`

Reply:
Sure, we will merge once you do that

Reply:
Done [changes](https://github.com/microsoft/rural-crowdsourcing-toolkit-client/compare/6e3378f437c54600173594ce09dd687345ba3b2c..e9ac0132246f6ba576720bff7d684574425113f9)